manifests/cluster-manager/management/cluster-manager-manifestworkreplicaset-deployment.yaml

```diff
@@ -92,16 +97,28 @@ spec:
         volumeMounts:
         - name: tmpdir
           mountPath: /tmp
+        {{ if eq .WorkDriver "kube" }}
```

Comment:
same question: 
hostedMode only in the case `workDirver == kube`?

Reply:
same as above.